src/main/java/io/vertx/json/schema/JsonFormatValidator.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2011-2020 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+package io.vertx.json.schema;
+
```

Comment:
I think this class must be annotated with `VertxGen`, as we are using it in `JsonSchemaOptions`, which is annotated with `DataObject`.